starter-code/routes/celebrities.js

```diff
@@ -0,0 +1,78 @@
+const express = require('express');
+const router  = express.Router();
+const CelebModel = require('../models/celeb.model')
+const bodyParser = require('body-parser');
```

Comment:
no need to require bodyParser here or in the movies route, this is already taken care of in app.js